lib/common/commonUtil.py

```diff
@@ -36,12 +55,199 @@ def record_status(self, case_name, status, value):
             json.dump(self.current_status, fh)
 
 
+class CalculationUtil(object):
```

Comment:
Should we re-write the `outlier` base on `CalculationUtil`?

Ref:
https://github.com/Mozilla-TWQA/Hasal/blob/dev/lib/common/outlier.py#L5
https://github.com/Mozilla-TWQA/Hasal/blob/dev/lib/helper/resultHelper.py#L128-L130
https://github.com/Mozilla-TWQA/Hasal/blob/dev/lib/helper/generatorHelper.py#L188-L190
https://github.com/Mozilla-TWQA/Hasal/blob/dev/server/server.py#L666